gensim/models/fasttext.py

```diff
@@ -754,7 +761,7 @@ def __contains__(self, word):
         return self.wv.__contains__(word)
 
     @classmethod
-    def load_fasttext_format(cls, model_file, encoding='utf8'):
+    def load_fasttext_format(cls, model_file, encoding='utf8', fast=False):
```

Comment:
I guess this should be named a bit different: `full_model` for example, `fast` is very multivalued (here and everywhere)